grails-app/domain/com/unifina/domain/data/Stream.groovy

```diff
@@ -62,6 +66,7 @@ class Stream implements Comparable {
 		requireSignedData defaultValue: "false"
 		autoConfigure defaultValue: "true"
 		storageDays defaultValue: DEFAULT_STORAGE_DAYS
+		exampleType enumType: "identity", defaultValue: ExampleType.NOT_SET
```

Comment:
Queries to fetch the examples can become slow if there are many rows. Should we add an index for this field?

Reply:
Excellent point!